include/CLI/TypeTools.hpp

```diff
@@ -968,6 +968,9 @@ bool integral_conversion(const std::string &input, T &output) noexcept {
         nstring.erase(std::remove(nstring.begin(), nstring.end(), '\''), nstring.end());
         return integral_conversion(nstring, output);
     }
+    if(std::isspace(static_cast<unsigned char>(input.back()))) {
+        return integral_conversion(trim_copy(input), output);
```

Comment:
Not covered by tests?

Reply:
missed that section on first try,  adding now.